USER-TRAINING.md

```diff
@@ -2,13 +2,18 @@
 
 This document is for Ops team members setting up new developers to work in the GPII cloud.
 
+## Introduction
+
+* Ask the user about their experience with infrastructure, command-line tools, cloud computing, etc.
+   * Let their responses guide how much you delve into or gloss over fine details, how much background information might be helpful, etc.
+
 ## The Speech
 
 * You are about to be granted administrative (aka "root") access to a number of resources on the public internet, including our staging and production environments. This is a serious responsiblity! You will be able to do harm to the system!
 * Measure twice, cut once.
 * If you're ever in doubt about what a command will do, STOP! Then ask for help.
-   * Ask other developers who have experience using gpii-infra
-   * Ask #ops in Slack
+   1. First, ask other developers who have experience using gpii-infra
```

Comment:
Markdown allows it, but that forces humans to renumber lists when adding or removing items.

It's a fair point about readability but I think rendered markdown will be consumed more frequently than raw markdown.

Reply:
To my first point, I forgot to say that while I understand why you say this could be more readable:

```
1. foo
1. bar
1. baz
1. qux
```

I find it more less distracting than:
```
1. foo
3. bar
4. baz
4. qux
```

:)

Reply:
Great point about the renumbering! I do like the pragmatism of this approach. 👍